README.md

```diff
@@ -1,57 +1,59 @@
-<a href="https://handsontable.github.io/hyperformula/">
-<img src="./github-hf-logo-blue.svg" width="250" height="51" alt="HyperFormula"/>
-</a>
-
-### HyperFormula is an open source, spreadsheet-like calculation engine ⚡
-
-[![GitHub Workflow Status](https://img.shields.io/github/workflow/status/handsontable/hyperformula/Test)](https://github.com/handsontable/hyperformula/actions?query=workflow%3ATest+branch%3Amaster)
-[![FOSSA Status](https://app.fossa.io/api/projects/git%2Bgithub.com%2Fhandsontable%2Fhyperformula.svg?type=shield)](https://app.fossa.io/projects/git%2Bgithub.com%2Fhandsontable%2Fhyperformula?ref=badge_shield)
-[![codecov](https://codecov.io/gh/handsontable/hyperformula/branch/master/graph/badge.svg?token=5k9ZQv8azO)](https://codecov.io/gh/handsontable/hyperformula)
-[![Language grade: JavaScript](https://img.shields.io/lgtm/grade/javascript/g/handsontable/hyperformula.svg?logo=lgtm&logoWidth=18)](https://lgtm.com/projects/g/handsontable/hyperformula/context:javascript)
-[![Known Vulnerabilities](https://snyk.io/test/github/handsontable/hyperformula/badge.svg?targetFile=package.json)](https://snyk.io/test/github/handsontable/hyperformula?targetFile=package.json)
-[![GitHub contributors](https://img.shields.io/github/contributors/handsontable/hyperformula)](https://github.com/handsontable/hyperformula/graphs/contributors)
-
-
-## Description
-
-HyperFormula allows you to perform spreadsheet-like calculations in your
-web applications. It is written in TypeScript and supports all major
-JavaScript frameworks. You can use it in a browser or as a service with
-Node.js as your back-end technology.
-
-The engine comes with a built-in, localized
-[library of functions](https://handsontable.github.io/hyperformula/guide/built-in-functions.html)
-grouped into different categories, and covers most user-triggered actions
-such as [CRUD operations](https://handsontable.github.io/hyperformula/guide/basic-operations.html),
-[undo/redo](https://handsontable.github.io/hyperformula/guide/undo-redo.html),
-and [clipboard operations](https://handsontable.github.io/hyperformula/guide/clipboard-operations.html).
-It also supports the use of cross-sheet references,
-[named expressions](https://handsontable.github.io/hyperformula/guide/named-expressions.html),
-different [data types](https://handsontable.github.io/hyperformula/guide/types-of-values.html),
-and [custom functions](https://handsontable.github.io/hyperformula/guide/custom-functions.html).
+<br>
+<p align="center">
+  <a href="https://handsontable.github.io/hyperformula/">
+    <img src="https://raw.githubusercontent.com/handsontable/hyperformula/master/github-hf-logo-blue.svg" width="350" height="71" alt="HyperFormula - An advanced headless evaluator and parser of the Excel formulas"/>
+  </a>
+</p>
+
+<p align="center">
+  <strong>An advanced headless evaluator and parser of the Excel formulas</strong>
```

Comment:
How about:

```suggestion
  <strong>A headless spreadsheet, an evaluator and parser of Excel formulas</strong>
```

Reply:
I changed this to:
```
<strong>A headless spreadsheet, a parser and evaluator of Excel formulas</strong>
```

Writing `parser and evaluator` (in this order) makes more sense because you always parse a formula first before evaluating it.

Reply:
Sure!